tests/integration/components/login-test.js

```diff
@@ -0,0 +1,45 @@
+import { module, test } from 'qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { setupRenderingTest } from 'website-www/tests/helpers';
+import { APPS } from '../../constants/urls';
+
+module('Integration | Component | login', function (hooks) {
```

Comment:
Tests for when clicking on the button? and other tests missing?

Reply:
i'll write the one for button click, what other tests do you suggest?